lib/Config/LocationAwareConfigRepository.php

```diff
@@ -227,12 +241,7 @@ public function isWriteable(?string $key = null, ?string $dataSource = null): bo
      */
     public function getWriteTarget(): string
     {
-        $env = $this->writeTargetEnvVariableName ? $_SERVER[$this->writeTargetEnvVariableName] ?? null : null;
-        if ($env) {
-            $writeLocation = $env;
-        } else {
-            $writeLocation = $this->defaultWriteLocation;
-        }
+        $writeLocation = $this->writeTarget ?? $this->defaultWriteLocation;
```

Comment:
@robertSt7 This is a BC break, right? Now all env variables havn't an effect. See failing tests in data-hub https://github.com/pimcore/data-hub/pull/702

Reply:
I see you read now the env variable direct in the DAO files. But I think this is still a BC break, all external DAOs must also be changed.

Reply:
@blankse fixed by https://github.com/pimcore/pimcore/pull/14486